dbld/images/helpers/packages.manifest

```diff
@@ -250,6 +254,7 @@ lsof                            [devshell]
 netcat                          [devshell]
 python-dbg                      [devshell]
 python3-dbg                     [devshell]
+snmptrapd                       [devshell]
```

Comment:
`snmptrapd` is a new dependency of Light. As other dependencies of Light are installed for basic platform images, I would install it also for them, for example:
```
# pip packages for python functional tests
pathlib2                [centos, fedora, debian, ubuntu]
psutil                  [centos, fedora, debian, ubuntu]
pytest                  [centos, fedora, debian, ubuntu]
```